examples/RodContactCase/RodSelfContact/PlectonemesCase/plectoneme_case.py

```diff
@@ -0,0 +1,218 @@
+import sys
+
+sys.path.append("../../../../")
+from elastica import *
+from examples.RodContactCase.post_processing import (
+    plot_video_with_surface,
+    plot_velocity,
+)
+
+
+class PlectonemesCase(
```

Comment:
Is self-contact enabled by default? Is there an option to turn it off?

Reply:
It is not enabled by default. User has to add it.

Reply:
@armantekinalp Thanks. See my comment on the new review below (I messed up the reply).